scrapylib/crawlera.py

```diff
@@ -28,6 +28,10 @@ class CrawleraMiddleware(object):
     ]
 
     def __init__(self, crawler):
+        warnings.warn(
+            'This version of CrawleraMiddleware is deprecated, '
+            'please use the version found in the scrapy-crawlera '
+            'package instead.')
```

Comment:
scrapylib/hubproxy.py needs to be updated too, right now it suggest to use `scrapylib.crawlera.CrawleraMiddleware` instead of `scrapy_crawlera.CrawleraMiddleware`, see https://github.com/scrapinghub/scrapylib/blob/master/scrapylib/hubproxy.py#L10